studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx

```diff
@@ -134,11 +137,21 @@ const GitHubIntegrationConnectionForm = ({
         <p className="text-xs text-light mb-3">
           All other branches will be treated as Preview branches
         </p>
+
+        <Alert_Shadcn_ className="mb-4 w-96">
+          <AlertTitle_Shadcn_ className="text-sm">
+            We currently don’t allow changing production branch
```

Comment:
```suggestion
            Changing Production branch coming soon
```

Reply:
opting for `Changing Git branch for production branch coming soon`?
"branching production branch" sounds a little odd to me haha